docs/docs/data-sources/Website.md

```diff
@@ -0,0 +1,32 @@
+Website data sources enrich your bot's responses by connecting to available online information. If your website contains relevant content, you can utilise it to enhance your bot's answers. We have have the capability to gather data from websites that can be searched using Bing, and then scrape this data. This collected information can be used to address inquiries and provide direct links to it.
+
+Algomo has the capacity to extract data from thousands of pages on a website, ensuring your visitors have access to the information they require.
+This data source allows you to re-scrape the website whenever necessary to keep your data up to date.
+
+There are two modes to website data sources:
+
+- **Scrape** - This option allows you to scrape the website and extract data from it.
```

Comment:
`This option allows you to save the information on your website pages within Algomo.`